pyne/partisn.py

```diff
@@ -233,7 +233,7 @@ def _get_material_lib(hdf5, data_hdf5path, nuc_hdf5path, **kwargs):
         comp = mats_collapsed[mat_name]
         atom_dens_dict = comp.to_atom_dens()
         comp_list = {}
-        for nucid, dens in atom_dens_dict.iteritems():
+        for nucid, dens in atom_dens_dict.items():
```

Comment:
A good Python3 change here...

Reply:
This was the minimum necessary to get the tests to pass on Py3.  Since there are many other instances of `iteritems()` that don't need to change for CI on Py3, it suggests dubious code coverage... but that's a separate issue.